components/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/IdentityProviderData.java

```diff
@@ -1,166 +0,0 @@
-/*
- * Copyright (c) 2005-2006, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- *  WSO2 Inc. licenses this file to you under the Apache License,
- *  Version 2.0 (the "License"); you may not use this file except
- *  in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.wso2.carbon.identity.provider;
-
-import org.apache.axiom.om.OMElement;
-import org.apache.axis2.context.MessageContext;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-import org.apache.rahas.RahasData;
-import org.apache.ws.security.WSConstants;
-import org.apache.ws.security.WSSecurityEngineResult;
-import org.apache.ws.security.handler.WSHandlerConstants;
-import org.apache.ws.security.handler.WSHandlerResult;
-import org.opensaml.SAMLAssertion;
-import org.wso2.carbon.identity.base.IdentityConstants;
-import org.wso2.carbon.user.core.claim.Claim;
-import org.wso2.carbon.utils.multitenancy.MultitenantUtils;
-
-import javax.xml.namespace.QName;
-import java.util.Vector;
-
-/**
- * Meta-data collection of related to CardSpace required for token issuance.
- */
-public class IdentityProviderData extends GenericIdentityProviderData {
-    private static final Log log = LogFactory.getLog(IdentityProviderData.class);
-    private SAMLAssertion assertion;
-
-    public IdentityProviderData(RahasData data) throws IdentityProviderException, ClassNotFoundException {
-        super(data);
-    }
-
-    /**
-     * @param rst
-     * @throws IdentityProviderException
-     */
-    @Override
-    protected void processInfoCardReference(OMElement rst) throws IdentityProviderException {
```

Comment:
Are these all the removed classes used for sts functionalities?

Reply:
Yeah but I had certain problems with IdentityProviderUtil since it had many methods but only one or two methods were used and that was also in STS implementation so moved into inbound-auth-sts and then TokenDecrypter which was not used anywhere(I deleted this since it was not used).

Reply:
All the OpenID integration tests passed without problems after this fix.